app/src/main/res/xml/device_filter.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please move this to app/src/visionglass/res directory as this is something very specific to vision glass